config/config.example.yml

```diff
@@ -23,6 +23,8 @@ services:
   Magpie:
     active: true
     url: https://${HOSTNAME}/magpie
+    admin_user: ${MAGPIE_ADMIN_USER}
+    admin_password: ${MAGPIE_ADMIN_PASSWORD}
```

Comment:
I followed what had be done for Geoserver's variable. These variables are set in the `cowbird.env` file when cowbird is used as a docker container : 
https://github.com/Ouranosinc/cowbird/blob/4bd9d5a3b0be8fcb9f99268189f1bfded50a4c06/docker/config/cowbird/cowbird.env#L6-L7
https://github.com/Ouranosinc/cowbird/blob/4bd9d5a3b0be8fcb9f99268189f1bfded50a4c06/docker/docker-compose.override.example.yml#L12-L13

I'm not sure it is working if we use a local cowbird instance though. I think we would have to set those manually or in another way, but I haven't really tried using a local setup. Tests are working since these values are defined in the `setup` functions. Do you think we should adapt the use of those variables? Or maybe this issue can be improved in a future PR?

Reply:
I looked more deeply into the code. Environnement variables are expanded when loading the YAML config by cowbird, so nothing more needs to be done.